Content.Shared/Pulling/Systems/SharedPullerSystem.cs

```diff
@@ -61,8 +62,11 @@ private void OnVirtualItemDeleted(EntityUid uid, SharedPullerComponent component
             if (args.Puller.Owner != uid)
                 return;
 
-            if (EntityManager.TryGetComponent(component.Owner, out SharedAlertsComponent? alerts))
-                alerts.ClearAlert(AlertType.Pulling);
+            if (EntityManager.TryGetComponent(component.Owner, out AlertsComponent? alerts))
+            {
+                var euid = alerts.Owner;
+                _alertsSystem.ClearAlert(euid, AlertType.Pulling);
+            }
```

Comment:
As neither `ClearAlert` nor `ShowAlert` take `AlertsComponent` as an argument, the old `TryGet` is completely unnecessary. Maybe just call `ClearAlert` directly w/o the `TryGet` (like you've done in other systems)?

Reply:
Missed one, thanks.